lightly/active_learning/agents/agent.py

```diff
@@ -14,14 +14,18 @@ class ActiveLearningAgent:
     Attributes:
         api_workflow_client:
             The client to connect to the api.
-        preselected_tag_id:
-            The id of the tag containing the already labeled samples, default: None == no labeled samples yet.
-        query_tag_id:
-            The id of the tag defining where to sample from, default: None resolves to initial_tag
+        query_set:
+            Set of filenames corresponding to samples which can possibly be selected.
+            Set to all samples in the query tag or to the whole dataset by default.
         labeled_set:
-            The filenames of the samples in the labeled set, List[str]
+            Set of filenames corresponding to samples in the labeled set.
+            Set to all samples in the preselected tag or to an empty list by default.
         unlabeled_set:
-            The filenames of the samples in the unlabeled set, List[str]
+            Set of filenames corresponding to samples which are in the query set
+            but not in the labeled set.
+        added_set:
+            Set of filenames corresponding to samples which were added to the 
+            labeled set in the last query.
```

Comment:
Optional: "added to the labeled set and removed from the unlabeled set in the last query"